x-pack/plugins/apm/public/components/app/ServiceDetails/index.tsx

```diff
@@ -26,19 +26,18 @@ export function ServiceDetails({ tab }: Props) {
   const plugin = useApmPluginContext();
   const { urlParams } = useUrlParams();
   const { serviceName } = urlParams;
-
-  const canReadAlerts = !!plugin.core.application.capabilities.apm[
-    'alerting:show'
-  ];
-  const canSaveAlerts = !!plugin.core.application.capabilities.apm[
-    'alerting:save'
-  ];
+  const capabilities = plugin.core.application.capabilities;
+  const canReadAlerts = !!capabilities.apm['alerting:show'];
+  const canSaveAlerts = !!capabilities.apm['alerting:save'];
   const isAlertingPluginEnabled = 'alerts' in plugin.plugins;
-
   const isAlertingAvailable =
     isAlertingPluginEnabled && (canReadAlerts || canSaveAlerts);
-
-  const { core } = useApmPluginContext();
+  const isMlPluginEnabled = 'ml' in plugin.plugins;
+  const canReadAnomalies = !!(
+    isMlPluginEnabled &&
+    capabilities.ml.canAccessML &&
+    capabilities.ml.canGetJobs
+  );
```

Comment:
nit: looks like all three are booleans so there should be no reason to coerce with `!!`

Reply:
I added the double negative because the logical and expression returns the type of the last operand (`capabilities.ml.canGetJobs`) which is: `boolean | Readonly<{[x: string]: boolean;}>`. Without the double NOT operator, `canReadAnomalies` fails the type assignment for the `canReadAnomalies` prop of the `AlertIntegrations` component.